CONTRIBUTING.md

```diff
@@ -34,14 +34,40 @@ If you haven't used `pipenv` before but are comfortable with virtualenvs, just r
 `pip install pipenv` in the virtualenv you're already using and invoke the command above
 from the cloned _Black_ repo. It will do the correct thing.
 
-Before submitting pull requests, run lints and tests with:
+Non pipenv install works too:
 
 ```console
+$ pip install -r test_requirements
+$ pip install -e .[d]
+```
+
+Before submitting pull requests, run lints and tests with the following commands from
+the root of the black repo:
+
+```console
+# Linting
 $ pre-commit run -a
-$ python -m unittest
+
+# Unit tests
+$ tox -e py
+
+# Optional Fuzz testing
+$ tox -e fuzz
+
+# Optional CI run to test your changes on many popular python projects
 $ black-primer [-k -w /tmp/black_test_repos]
 ```
 
+### Docs Testing
+
+If you make changes to docs, you can test they still build locally too.
```

Comment:
Technically this step is also needed when you make modifications to documentation files which are duplicated (for reasons I don't really know / understand). I honestly wish and might try to get these duplicated files and the regen step gone for good. It really doesn't matter if the duplicated content is up to date since they will updated before it get deploys on ReadTheDocs as a fresh build always starts, and IMO GitHub isn't the right place for documentation hosting. IIRC one of the reasons why we duplicate the docs is because it improves the GitHub documentation reading experience, but we have ReadTheDocs already, a platform *designed* for documentation.

Reply:
I +1 this cleanup. I think it's not worth having the duplicates too.

I'm just keeping my PRs to address/solve one main problem with each, as you noted.